conftest.py

```diff
@@ -15,9 +16,10 @@
 
 def pytest_report_header(config, startdir):
     """Add dependency information to pytest output."""
-    return ('Dependencies: Matplotlib ({}), NumPy ({}), '
+    return ('Dependencies: Matplotlib ({}), NumPy ({}), Pandas ({}), '
```

Comment:
Should we also add Xarray for consistency?

Reply:
Ah yeah, that would probably be good.